### PR TITLE
Use `electron-store` to check whether it's the first run

### PR DESCRIPTION
**Pull request by joecohens**
Closes https://github.com/zeit/now-desktop/issues/507

### CODE REVIEW

File: main/utils/first-run.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  opts = Object.assign({ defaults: { firstRun: true } }, opts)
+
+  return new Store(opts)
+}
+
+const firstRun = opts => {
+  const conf = getStore(opts)
+  let firstRun
+
+  if (firstRun === undefined) {
```
Comment (morajabi) [inline main/utils/first-run.js]: Could you please elaborate a bit on what this if statement is doing? :)

File: main/utils/first-run.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    firstRun = conf.get('firstRun')
+  }
+
+  if (firstRun === true) {
+    conf.set('firstRun', false)
+  }
+
+  return firstRun
```
Comment (morajabi) [inline main/utils/first-run.js]: Do you think it would be nice if we don't shadow the variable? 👍

Comment (morajabi) [inline main/utils/first-run.js]: Do you think it would be nice if we don't shadow the variable? 👍

Comment (joecohens) [reply main/utils/first-run.js]: I've cleaned up a little bit the implementation, thanks for your comments!

**Comment by leo** [comment]
@joecohens Thanks for the PR!

But can you switch it to `electron-store`, please? It was [renamed](https://github.com/sindresorhus/electron-store/issues/4) recently.

**Pull request merged**